src/registrar/config/settings.py

```diff
@@ -43,11 +43,8 @@
 DEBUG = env.bool("DJANGO_DEBUG", default=False)
 
 # TODO: configure and document security settings
-ALLOWED_HOSTS = [
-    'getgov-unstable.app.cloud.gov',
-    'get.gov'
-]
-ALLOWED_CIDR_NETS = ['10.0.0.0/8']  # nosec
+ALLOWED_HOSTS = ["getgov-unstable.app.cloud.gov", "get.gov"]
```

Comment:
I have discovered that this irritating* folding can be avoided by [use of a trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma).

Not something to fix here, as I am doing a bunch of work in this file, just sharing FYI.

\* I call it irritating because it breaks the fundamental code formatting (& design) principle of "similar things should look similar, different things should look different".